test/boost/sstable_datafile_test.cc

```diff
@@ -470,7 +470,7 @@ SEASTAR_TEST_CASE(datafile_generation_47) {
 
         auto sst = env.make_sstable(s, 47);
         write_memtable_to_sstable_for_test(*mt, sst).get();
-        auto sstp = env.reusable_sst(s, 47).get();
+        auto sstp = env.reusable_sst(s, sst).get();
```

Comment:
The `s` seem to be the same as for `sst`, why not call schema-less reusable_sst() overload?